lantern_embeddings_core/tests/text_embedding_test.rs

```diff
@@ -27,7 +47,11 @@ macro_rules! text_embedding_test {
 
             let expected_output: Vec<Vec<f32>> =
                 itertools::repeat_n(expected, batch_size).collect();
-            assert_eq!(output, expected_output);
+
+            for i in 0..expected_output.len() {
+                let similarity = 1.0 - cosine_similarity(&output[i], &expected_output[i]);
```

Comment:
`1 - consine_similarity` is cosine distance
so might make more sense to name this `let distance`. Then, the assertion makes more sense